src/core_plugins/elasticsearch/index.js

```diff
@@ -15,8 +16,6 @@ const DEFAULT_REQUEST_HEADERS = [ 'authorization' ];
 
 module.exports = function ({ Plugin }) {
   return new Plugin({
-    require: ['kibana'],
```

Comment:
I like that, I'll see what I can do.  It waits until after 5 synchronous network request, so it's technically possible and it would probably only be a matter of time before we made health check changes and things broke.